test/test.js

```diff
@@ -1710,11 +1596,9 @@ describe("Data directory cleanup", function() {
         sslEnabled: false,
         s3ForcePathStyle: true
       });
-      s3Client.createBucket({ Bucket: bucket }, function(err) {
-        if (err) return done(err);
-        generateTestObjects(s3Client, bucket, 10, function(err) {
-          if (err) return done(err);
-          s3rver.close(function(err) {
+      s3Client.createBucket({ Bucket: bucket }, err => {
+        generateTestObjects(s3Client, bucket, 10, () => {
```

Comment:
Why was `if (err) return done(err)` removed for these two calls? Two instances of this change below as well.